hie.yaml

```diff
@@ -0,0 +1,4 @@
+cradle:
+  bios:
+    program: script/ghci-flags
+    dependency-program: script/ghci-flags-dependencies
```

Comment:
1. `hie-bios` recently added support for multiple cabal component cradles, but I couldn’t get it working with `ghcide` yet.

2. My `direct` configuration wasn’t portable due to VS Code using a `PWD` env var of `/` and `-i` flags being resolved relative to that env var rather than the actual working directory. So we had to compute it.

3. Computing it is probably better for our purposes anyway, since it allows us to share work between this and `script/repl`.